includes/pojo-a11y-frontend.php

```diff
@@ -217,23 +217,23 @@ private function get_toolbar_svg( $icon ) {
+				'reset'             => '<path fill="currentColor" d="M384 224c0 105.75-86.25 192-192 192-57.25 0-111.25-25.25-147.75-69.25-2.5-3.25-2.25-8 .5-10.75L79 301.5c1.75-1.5 4-2.25 6.25-2.25 2.25.25 4.5 1.25 5.75 3C115.5 334 152.25 352 192 352c70.5 0 128-57.5 128-128S262.5 96 192 96c-32.75 0-63.75 12.5-87 34.25l34.25 34.5c4.75 4.5 6 11.5 3.5 17.25-2.5 6-8.25 10-14.75 10H16c-8.75 0-16-7.25-16-16V64c0-6.5 4-12.25 10-14.75 5.75-2.5 12.75-1.25 17.25 3.5L59.75 85C95 51.75 142.75 32 192 32c105.75 0 192 86.25 192 192z"/>',
+				'sitemap'           => '<path fill="currentColor" d="M448 312v80c0 13.25-10.75 24-24 24h-80c-13.25 0-24-10.75-24-24v-80c0-13.25 10.75-24 24-24h24v-48H240v48h24c13.25 0 24 10.75 24 24v80c0 13.25-10.75 24-24 24h-80c-13.25 0-24-10.75-24-24v-80c0-13.25 10.75-24 24-24h24v-48H80v48h24c13.25 0 24 10.75 24 24v80c0 13.25-10.75 24-24 24H24c-13.25 0-24-10.75-24-24v-80c0-13.25 10.75-24 24-24h24v-48c0-17.5 14.5-32 32-32h128v-48h-24c-13.25 0-24-10.75-24-24V56c0-13.25 10.75-24 24-24h80c13.25 0 24 10.75 24 24v80c0 13.25-10.75 24-24 24h-24v48h128c17.5 0 32 14.5 32 32v48h24c13.25 0 24 10.75 24 24z"/>',
+				'help'              => '<path fill="currentColor" d="M224 344v-48c0-4.5-3.5-8-8-8h-48c-4.5 0-8 3.5-8 8v48c0 4.5 3.5 8 8 8h48c4.5 0 8-3.5 8-8zm64-168c0-45.75-48-80-91-80-40.75 0-71.25 17.5-92.75 53.25-2.25 3.5-1.25 8 2 10.5l33 25c1.25 1 3 1.5 4.75 1.5 2.25 0 4.75-1 6.25-3 11.75-15 16.75-19.5 21.5-23 4.25-3 12.5-6 21.5-6 16 0 30.75 10.25 30.75 21.25 0 13-6.75 19.5-22 26.5-17.75 8-42 28.75-42 53v9c0 4.5 3.5 8 8 8h48c4.5 0 8-3.5 8-8 0-5.75 7.25-18 19-24.75C262 228.5 288 214 288 176zm96 48c0 106-86 192-192 192S0 330 0 224 86 32 192 32s192 86 192 192z"/>',
+				'feedback'          => '<path fill="currentColor" d="M448 224c0 88.5-100.25 160-224 160-12.25 0-24.5-.75-36.25-2-32.75 29-71.75 49.5-115 60.5-9 2.5-18.75 4.25-28.5 5.5-5.5.5-10.75-3.5-12-9.5v-.25c-1.25-6.25 3-10 6.75-14.5C54.75 406 72.75 391 84.5 349.25 33 320 0 274.75 0 224 0 135.75 100.25 64 224 64s224 71.5 224 160z"/>',
 			);
 		}
 
 		if ( isset( $icons[ $icon ] ) ) {
-			return sprintf( '<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="1em" viewBox="0 0 448 448">%s</svg>', $icons[ $icon ] );
+			return sprintf( '<svg xmlns="http://www.w3.org/2000/svg" width="1em" viewBox="0 0 448 448">%s</svg>', $icons[ $icon ] );
```

Comment:
Arguably, this should include a `height` attribute too. IE for example is known not to work properly without dimensions.